### ISSUE TITLE
MicroCeph: Test entry-level tutorial

### ISSUE THREAD

**Issue by skoech** [canonical/open-documentation-academy#174]
## Background

We recently created an entry-level [MicroCeph tutorial](https://canonical-microceph.readthedocs-hosted.com/en/latest/tutorial/get-started/) to serve new users of Ceph.

## Why is this needed

The tutorial's goal is to serve as an entry point for new users, to help them get started with MicroCeph and set them up for their next steps, whether it is to use MicroCeph to accomplish certain tasks in their work (how-to guides), learn more about MicroCeph (explanation) or make quick references.

It is intended to provide a complete, end-to-end learning experience. We have tried our best to create this experience, but we cannot do it alone. We would greatly benefit from user feedback- thus our request for enhancement/improvement. All feedback is valuable; you don't have to to have any prior experience with Ceph.

## What we need from you

We request the following of you:

1. Follow along the tutorial, testing all commands to see if they work as expected.
2. Record any anomalies and inconsistencies, from grammar to technical errors, and everything in between.
3. You can claim this issue here, but make sure to comment under [this issue in the MicroCeph repository](https://github.com/canonical/microceph/issues/514) with your feedback, whether negative or positive.

## Resources

- Diataxis: [tutorials](https://diataxis.fr/tutorials/).
- Check out the [Ubuntu Code of Conduct](https://ubuntu.com/community/ethos/code-of-conduct).
- If you are new to the Canonical Open Documentation Academy, you may also want to review the [Getting Started content](https://discourse.ubuntu.com/t/getting-started/42769).

Thank you in advance for your contribution!

**Comment by vpankitasingh** [comment]
@skoech I'd like to take this up! Could you please assign it to me?

**Comment by skoech** [comment]
Hi @vpankitasingh, sure thing! Thank you for your interest!

Please let me know if you need anything.

**Comment by vpankitasingh** [comment]
@skoech create a pull request https://github.com/canonical/microceph/pull/515

**Comment by skoech** [comment]
Thank you, @vpankitasingh. I'll go over and look at it now.

**Comment by lexbcarter** [comment]
@skoech Hello! Could I try to take this on, please?

**Comment by skoech** [comment]
@lexbcarter, thank you so much for your interest. @vpankitasingh has already done some work on this task, but multiple testers for our tutorial would actually be great.
I think it would be best to create a separate task for you to avoid conflicts. I'll let you know once I do.

**Comment by kellytrinh** [comment]
I am happy to help out to improve microceph on docs side (I am not a coder)

**Comment by skoech** [comment]
@kellytrinh, that's excellent! We would really appreciate your help.
I see that you are already actively participating in our community channel (even left a comment on a PR), and we are looking forward to more of that.
Please feel free to go over [our docs](https://canonical-microceph.readthedocs-hosted.com/en/latest/) and make any changes or suggestions. You can also look over our [issues list](https://github.com/canonical/microceph/issues) to see if there's anything you'd like to work on.
We have recently added a [contributing guide](https://canonical-microceph.readthedocs-hosted.com/en/latest/contributing/) as well, please refer to it if you're unsure of something.

**Comment by skoech** [comment]
@vpankitasingh, thank you so much for your excellent contribution.

I have merged your PR into the MicroCeph docs, and am closing this issue as complete now.
